### PR TITLE
Ready PR for test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/components/codeEditor/editor.jsx [hunk 1]
```diff
@@ -69,16 +105,53 @@ const Editor = ({ header, setHeader, footer, setFooter }) => {
...
+          </ul>
+          <span className='close-btn cursor-pointer' onClick={() => onHide()}>
+            <IoIosClose />
+          </span>
+        </div>
+        <div className='tab-content'>{renderTabContent()}</div>
+      </div>
+      <div className='btn-container w-100 my-3 d-flex align-items-center gap-3'>
+        <button
+          className='border rounded py-1 px-3 btn btn-outline-secondary ml-auto'
```
Comment (Saurabh186) [inline src/components/codeEditor/editor.jsx:146]: remove ml-auto and use justify-content-end in parent

File: src/components/codeEditor/editor.scss [hunk 2]
```diff
@@ -1,85 +1,117 @@
...
-    margin-bottom: 10px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-    width: 630px;
-  
-  }
-  
+}
+
+.preview {
+  border: #333;
```
Comment (Saurabh186) [inline src/components/codeEditor/editor.scss:11]: use variables

File: src/components/codeEditor/editor.scss [hunk 3]
```diff
@@ -1,85 +1,117 @@
...
+  border-radius: 10px;
```
Comment (Saurabh186) [inline src/components/codeEditor/editor.scss:12]: use rounded-2

File: src/components/codeEditor/editor.scss [hunk 4]
```diff
@@ -1,85 +1,117 @@
...
+}
+
+.box {
+  background-color: #f9f9f9;
```
Comment (Saurabh186) [inline src/components/codeEditor/editor.scss:16]: use variables

Comment (Saurabh186) [reply src/components/codeEditor/editor.scss:16]: yaha jo likhe kya class use nahi kar sakte kya inke ya run time me code ban ra hai

File: src/components/common/iconButton.jsx [hunk 5]
```diff
@@ -4,7 +4,7 @@ import './iconButton.scss'
 export default function IconButton(props) {
   return (
     <div onClick={props?.onClick} 
-    className={((props?.variant === 'sm' ? 'icon-button-sm cursor-pointer' : 'icon-button cursor-pointer') + ' ' + 'd-flex justify-content-center align-items-center') + ' ' + props?.className}>
+    className={((props?.variant === 'sm' ? 'icon-button-sm cursor-pointer' : 'icon-button cursor-pointer') + ' ' + 'd-flex justify-content-start align-items-center') + ' ' + props?.className}>
```
Comment (Saurabh186) [inline src/components/common/iconButton.jsx:7]: remove justify-content-start by default start rahta hai

File: src/components/endpoints/codeTemplate.jsx [hunk 6]
```diff
@@ -141,7 +141,7 @@ const CodeTemplate = (props) => {
 
   return (
     <>
-      <div className={`${isOnPublishedPage() ? 'position-sticky top-0 h-100' :'sample-code-inner position-sticky top-0'}`}>
+      <div className={`${isOnPublishedPage() ? 'position-sticky h-100' :'sample-code-inner position-sticky'}`}  style={{ top: '110px' }}>
```
Comment (Saurabh186) [inline src/components/endpoints/codeTemplate.jsx:144]: remove inline class and use class and write in scss file

File: src/components/endpoints/displayEndpoint.jsx [hunk 7]
```diff
@@ -3221,7 +3221,7 @@ class DisplayEndpoint extends Component {
         <div className={this.isNotDashboardOrDocView() ? 'mainContentWrapper dashboardPage' : 'mainContentWrapper d-flex'}>
           <div className={`innerContainer w-100 ${'response-bottom'}`}>
             <div
-              className={`hm-endpoint-container mid-part endpoint-container pt-0 ${!isOnPublishedPage() ? 'px-3' : ''} ${this.props?.endpointContent?.currentView === 'doc' ? 'doc-fix-width' : ''
+              className={`hm-endpoint-container mid-part endpoint-container pt-0 ${!isOnPublishedPage() ? 'px-3' : 'ps-5 pe-4'} ${this.props?.endpointContent?.currentView === 'doc' ? 'doc-fix-width' : ''
```
Comment (Saurabh186) [inline src/components/endpoints/displayEndpoint.jsx:3224]: bootstrap 5 update ho gaya h kya ps-5 pe-4 why r using

File: src/components/main/main.scss [hunk 8]
```diff
@@ -4363,4 +4358,19 @@ a {
...
+.hm-main-wrapper {
+  &:has(.navbar-public) {
+    .public-sidebar-container,
+    .heading-main {
+      top: 112px;
+    }
+  }
+}
+
+@media screen and (max-width: 768px) {
```
Comment (Saurabh186) [inline src/components/main/main.scss:4372]: glt hai isko .heading ke ander lagao

File: src/components/publicSidebar/smallScreenPublicSidebar/smallScreenPublicSidebar.jsx [hunk 9]
```diff
@@ -19,7 +19,7 @@ export default function SmallScreenPublicSidebar({ sidebarData }) {
 
     return (
         <>
-            <div className="small-screen-header w-100 bg-white d-none d-flex flex-column justify-content-start p-4 align-items-center">
+            <div className="small-screen-header w-100 bg-white d-flex flex-column justify-content-start py-4 pr-4 align-items-center">
```
Comment (Saurabh186) [inline src/components/publicSidebar/smallScreenPublicSidebar/smallScreenPublicSidebar.jsx:22]: remove justify-content-start

File: src/components/publishDocs/publishDocsForm.jsx [hunk 10]
```diff
@@ -519,39 +525,41 @@ const PublishDocForm = (props) => {
...
+                    </Dropdown.Menu>
+                  </Dropdown>
+                ) : (
+                  <input
+                    type='text'
+                    className='d-block w-100 font-12'
+                    value='Public Environment'
+                    readOnly
+                    onClick={() => handlePublicEnvClick()}
+                    style={{ cursor: 'pointer', border: '1px solid #ced4da', padding: '5px', borderRadius: '4px' }}
```
Comment (Saurabh186) [inline src/components/publishDocs/publishDocsForm.jsx:558]: remove inline classs and use classes and write in scss file

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/codeEditor/editor.jsx (hunk 1 above)
Comment (ViaSocket-Git) [reply src/components/codeEditor/editor.jsx:146]: Resolved

File: src/components/codeEditor/editor.scss (hunk 2 above)
Comment (ViaSocket-Git) [reply src/components/codeEditor/editor.scss:11]: Run Time

File: src/components/codeEditor/editor.scss (hunk 3 above)
Comment (ViaSocket-Git) [reply src/components/codeEditor/editor.scss:12]: Run Time

File: src/components/main/main.scss (hunk 8 above)
Comment (ViaSocket-Git) [reply src/components/main/main.scss:4372]: Resolved

File: src/components/endpoints/codeTemplate.jsx (hunk 6 above)
Comment (ViaSocket-Git) [reply src/components/endpoints/codeTemplate.jsx:144]: Resolved

File: src/components/publicSidebar/smallScreenPublicSidebar/smallScreenPublicSidebar.jsx (hunk 9 above)
Comment (ViaSocket-Git) [reply src/components/publicSidebar/smallScreenPublicSidebar/smallScreenPublicSidebar.jsx:22]: Resolved

File: src/components/common/iconButton.jsx (hunk 5 above)
Comment (ViaSocket-Git) [reply src/components/common/iconButton.jsx:7]: Resolved

File: src/components/publishDocs/publishDocsForm.jsx (hunk 10 above)
Comment (ViaSocket-Git) [reply src/components/publishDocs/publishDocsForm.jsx:558]: Resolved

**Review by Saurabh186 — APPROVED** [review]

**Pull request merged**